추가: Prisma 구축 및 로그인 구현

kkw01234 opened the pull request:
Prisma 구축하고
Front에서 넘어온 accessToken을 이용하여 유저 여부 판단 후 유저DB에 추가/수정하고 새로운 토큰을 발급하여 이를 이용해서 권한을 부여하는 방식을 구현

kkw01234 committed:
추가: 서버에서 토큰을 받고 그에 따라 Header 뷰 변환 구현

비로그인시와 로그인시 헤더를 다르게하기 위해서 구현

kkw01234 committed:
추가: 로그인시 드롭다운으로 마이페이지, 로그아웃 띄우기

드롭다운을 이용하여 다른페이지로 넘어갈수 있게 구현, 로그아웃도할 수
있게 구현

kkw01234 committed:
추가: 프리즈마를 사용하여 데이터베이스 접속하기 위해서 구축

프리즈마를 ORM으로 사용하고 데이터 모델링까지 진행

kkw01234 committed:
추가: accessToken일 때 user 데이터 저장 및 토큰 생성

accessToken으로 OAuth 유저 확인 후 새로운 토큰을 발급하여 새로운토큰으로
권한을 부여하기 위해서 구현

kkw01234 committed:
추가: Prisma 구축 및 로그인 구현

Prisma 구축하고
Front에서 넘어온 accessToken을 이용하여 유저 여부 판단 후 유저DB에
추가/수정하고 새로운 토큰을 발급하여 이를 이용해서 권한을 부여하는
방식을 구현

kkw01234 committed:
기타: 컴플릭트 난 것 변경

munsulim commented on back/prisma.yml:
```diff
@@ -0,0 +1,6 @@
+endpoint: http://210.89.189.122:4466
```
endpoint env로 숨겨주세요.

kkw01234 committed:
추가: 프리즈마를 사용하여 데이터베이스 접속하기 위해서 구축

프리즈마를 ORM으로 사용하고 데이터 모델링까지 진행

kkw01234 committed:
추가: accessToken일 때 user 데이터 저장 및 토큰 생성

accessToken으로 OAuth 유저 확인 후 새로운 토큰을 발급하여 새로운토큰으로
권한을 부여하기 위해서 구현

kkw01234 committed:
Merge branch 'feature/OAuthBack' of https://github.com/connect-foundation/2019-13 into feature/OAuthBack

kkw01234 committed:
기타: 필요없는 파일 삭제

kkw01234 committed:
추가: Prisma 구축 및 로그인 구현

Prisma 구축하고
Front에서 넘어온 accessToken을 이용하여 유저 여부 판단 후 유저DB에
추가/수정하고 새로운 토큰을 발급하여 이를 이용해서 권한을 부여하는
방식을 구현

kkw01234 committed:
Merge branch 'feature/OAuthBack' of https://github.com/connect-foundation/2019-13 into feature/OAuthBack

kimjouny commented:
확인함
